site/data/data.json

```diff
@@ -13648,6 +13822,21 @@
       }
     ]
   },
+  {
+    "name": "Merry Arts, Lakewood Village Tavern, Academy Tavern",
```

Comment:
@vsalvino is this a suspect entry? I hid it, added carl to the Lakewood village entry

Reply:
Some people try to list multiple restaurants, even though we ask them not to. I usually just take the first one and chop off the rest.